src/components/Transactions/TransactionStatus.tsx

```diff
@@ -0,0 +1,51 @@
+import { CheckCircleIcon, WarningTwoIcon } from '@chakra-ui/icons'
+import { Tag } from '@chakra-ui/react'
+import { chainAdapters } from '@shapeshiftoss/types'
+import { CircularProgress } from 'components/CircularProgress/CircularProgress'
+import { Row } from 'components/Row/Row'
+import { Text } from 'components/Text'
+
+export const TransactionStatus = ({ txStatus }: { txStatus: chainAdapters.TxStatus }) => {
+  const status = (() => {
+    switch (txStatus) {
+      case chainAdapters.TxStatus.Confirmed:
+        return (
+          <>
+            <Tag colorScheme='green' size='lg'>
+              <CheckCircleIcon mr={2} />
+              <Text translation='transactionRow.confirmed' />
+            </Tag>
+          </>
+        )
+      case chainAdapters.TxStatus.Pending:
+        return (
+          <>
+            <Tag colorScheme='blue' size='lg'>
+              <CircularProgress mr={2} size='5' />
+              <Text translation='transactionRow.pending' />
+            </Tag>
+          </>
+        )
+      default:
```

Comment:
Why not be exhaustive and go with `case chainAdapters.TxStatus.Failed` here?

Reply:
Yeah true, we could do a fall-through: https://github.com/shapeshift/web/pull/1094/commits/d2150eda4c5f707429e3c2dcba9d2444a640b853